### PR TITLE
AC Automated Test - LRAC-10621 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10621 (Group)
https://issues.liferay.com/browse/LRAC-10622 (Automation ticket)
https://issues.liferay.com/browse/LRAC-8720 (Story)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by r4ddek** [comment]
Hi @ViniLopes20 , the code looks good to me. As a suggestion, I'd change the name of the `ACUtils.viewGenericText` macro to `ACUtils.assertGenericText` to better reflect its function in the name.

**Review by r4ddek — APPROVED** [review]

**Review by manuferreira — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Onboarding.testcase [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        task ("Copy the token from AC") {
+			Copy(locator1 = "ACSettings#TOKEN_INPUT_FIELD");
+		}
+
+        task ("Go to DXP and connect the DXP to AC") {
+            ACUtils.launchDXP();
+
+			ACDXPSettings.connectDXPtoAnalyticsCloud();
+		}
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Onboarding.testcase:93]: hi @ViniLopes20 , what do you think if we add a comment explaining what we are going to do next?
For example: `// Now we are going to add a new DXP Data Source and validate the info shown in the connection modal from Settings.`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Onboarding.testcase [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Check the onboarding connection modal information") {
+			task ("Check if the modal title and the copy token button are present") {
+				ACUtils.viewGenericText(textValueList = "Connect Your DXP Analytics");
+
+            	AssertElementPresent(locator1 = "ACSettings#COPY_TOKEN_BUTTON");
+			}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Onboarding.testcase:107]: hi @ViniLopes20 , I know the modal has a class called "`onboarding...`" but I think to make things clear here, we could have this task renamed to:
`Check the connection modal information when opened in Settings`

What do you think?

**Comment by alexcandidos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Review by alexcandidos — APPROVED** [review]

**Pull request closed without merging**